src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1320,5 +1320,6 @@ internal enum ErrorCode
         ERR_InvalidPathMap = 8101,
         ERR_PublicSignButNoKey = 8102,
         ERR_TooManyUserStrings = 8103,
+        ERR_PEWritingFailure = 8104,
```

Comment:
.NET naming convention uses lower case letters for abbreviations after the first letter, so it should be `PeWritingFailure`